ftw/lawgiver/wdl/specification.py

```diff
@@ -90,6 +90,7 @@ def __init__(self, title, src_status=None, dest_status=None,
         self._src_status_title = src_status_title or src_status.title
         self.dest_status = dest_status
         self._dest_status_title = dest_status_title or dest_status.title
+        self.options = options
 
```

Comment:
The specification should also validate the input. In this case I'd expect it to detect when a make a typo (e.g. `guard_expr`) or when I use an unknown key (e.g. when I use the newest `ftw.lawgiver` docu but using an older version where this key does not yet exist).

I would do that with:
```python
unsupported_options = tuple(set(self.options) - {'guard-expression'})
if len(unsupported_options) == 0:
  raise ValueError('Unkown transition options: {!r}'.format(unsupported_options))
```